backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -152,6 +157,8 @@ public TestOrder addTestResult(
     order.setTestEventRef(testEvent);
     TestOrder savedOrder = _repo.save(order);
 
+    _testEventReportingService.report(testEvent);
```

Comment:
I realize that the rest of this method is synchronous, but is there any reason _not_ to use the async call from the TestEventReportingService here?

Reply:
I need the overall transaction to fail and roll back if the queue doesn't accept the message, so the result of `reportAsync` would need to be joined & unwrapped here in any case.

I think we could use the async version to run this in parallel with the SMS dispatch.